src/syscheckd/syscheck_audit.c

```diff
@@ -232,7 +232,7 @@ int add_audit_rules_syscheck(bool first_time) {
                         w_mutex_unlock(&audit_rules_mutex);
                         rules_added++;
                     } else {
-                        merror(FIM_ERROR_WHODATA_ADD_RULE, retval, syscheck.dir[i]);
+                        mwarn(FIM_WARN_WHODATA_ADD_RULE, syscheck.dir[i]);
```

Comment:
This message appears every time the audit rules are reloaded (30 seconds by default).

It would be interesting to show it as a warning for the first time only. After that, it can be passed to debug1.